analysed-packages/logrotate/version-3.20.1/logrotate-3.20.1-OSS-disclosure.txt

```diff
@@ -0,0 +1,2076 @@
+you are welcome to redistribute it under certain conditions; type `show c' for
+details.
+
+The hypothetical commands `show w' and `show c' should show the appropriate
+parts of the General Public License. Of course, the commands you use may be
+called something other than `show w' and `show c'; they could even be mouse-
+clicks or menu items--whatever suits your program.
+
+You should also get your employer (if you work as a programmer) or your school,
+if any, to sign a "copyright disclaimer" for the program, if necessary. Here is
+a sample; alter the names:
+
+Yoyodyne, Inc., hereby disclaims all copyright interest in the program
+`Gnomovision' (which makes passes at compilers) written by James Hacker.
+
+<signature of Ty Coon>, 1 April 1989 Ty Coon, President of Vice
+
+This General Public License does not permit incorporating your program into
+proprietary programs. If your program is a subroutine library, you may consider
+it more useful to permit linking proprietary applications with the library. If
+this is what you want to do, use the GNU Lesser General Public License instead
+of this License.
+
+License text of the Autoconf-exception-generic as an exception to the
+GPL-2.0-or-later:
+
+As a special exception to the GNU General Public License, if you distribute this
+file as part of a program that contains a configuration script generated by
+Autoconf, you may include it under the same distribution terms that you use for
+the rest of that program.
+
+--------------------------------------------------------------------------------
+
+License-of-GNU-Licenses
+
+Everyone is permitted to copy and distribute verbatim copies
+of this license document, but changing it is not allowed.
+
+--------------------------------------------------------------------------------
+
+FSFULLRWD
+
+This Makefile.in is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+This program is distributed in the hope that it will be useful, but WITHOUT ANY
+WARRANTY, to the extent permitted by law; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+
+--------------------------------------------------------------------------------
+
+FSFULLR
+
+This file is free software; the Free Software Foundation gives unlimited
+permission to copy and/or distribute it, with or without modifications, as long
+as this notice is preserved.
+
+--------------------------------------------------------------------------------
+
+FSFUL
+
+This configure script is free software; the Free Software Foundation gives
+unlimited permission to copy, distribute and modify it.
+
+--------------------------------------------------------------------------------
+
+Public-domain
+
+Public domain
+
+--------------------------------------------------------------------------------
+
+BSD-3-Clause
+
+Copyright (c) <year> <owner>.
```

Comment:
I think that this is not the correct license text. The BSD-3-Clause is the license of queue.h and in queue.h there is the following license text:
Redistribution and use in source and binary forms, with or without
modification, are permitted provided that the following conditions
are met:
1. Redistributions of source code must retain the above copyright
notice, this list of conditions and the following disclaimer.
2. Redistributions in binary form must reproduce the above copyright
notice, this list of conditions and the following disclaimer in the
documentation and/or other materials provided with the distribution.
3. Neither the name of the University nor the names of its contributors
may be used to endorse or promote products derived from this software
without specific prior written permission.

THIS SOFTWARE IS PROVIDED BY THE REGENTS AND CONTRIBUTORS ``AS IS'' AND
ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE
ARE DISCLAIMED. IN NO EVENT SHALL THE REGENTS OR CONTRIBUTORS BE LIABLE
FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS
OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION)
HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT
LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY
OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF
SUCH DAMAGE.
Can you please correct this